build.cake

```diff
@@ -71,7 +71,7 @@ Task("ResolveBuildTools")
     var vsLatest = VSWhereLatest(vsWhereSettings);
     msBuildPath = (vsLatest == null)
         ? null
-        : vsLatest.CombineWithFilePath("./MSBuild/15.0/Bin/MSBuild.exe");
+        : vsLatest.CombineWithFilePath("./MSBuild/15.0/Bin/amd64/MSBuild.exe");
```

Comment:
Previously there was an issue with the amd64 stuff not being able to locate `csc.exe`. Not sure if that is fixed. Just FYI.

Reply:
no that's still an issue

Reply:
Amazing :)